docs/compositions.md

```diff
@@ -0,0 +1,144 @@
+# Action compositions
+
+Action compositions make it possible to dynamically build and invoke a series of actions. Action compositions are similar to sequences. However, whereas the components of a sequence must be specified before invoking the sequence, components of a composition can be decided as the composition is running.
+
+## Example
+
+Suppose we define an _increment_ action:
```

Comment:
Suppose we define an `_increment_ action`:

Reply:
I am missing the diff here...

Reply:
So what is the story for [a|A]ction [c|C]omposition and the [c|C]omposition shorthand?